api/nfd/v1alpha1/types.go

```diff
@@ -112,7 +112,7 @@ type NodeFeatureRuleList struct {
 }
 
 // NodeFeatureRule resource specifies a configuration for feature-based
-// customization of node objects, such as node labeling.
+// customization of node objects, such as node labeling. LOOSEME
```

Comment:
LOOSEME?

Reply:
ups that was my hack to check things were being updated properly